### PR TITLE
Aoc/456 placeholder labels change

### PR DESCRIPTION
**Pull request by chengoconnell**
Screenshot 2022-06-29 at 17 13 10

## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Review by sintinilorenzo — APPROVED** [review]

File: packages/slice-machine/lib/forms/defaults.ts [hunk 1]
```diff
@@ -18,7 +18,16 @@ export const validateId = ({
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const DefaultFields: any = {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
```
Comment (MarcMcIntosh) [inline packages/slice-machine/lib/forms/defaults.ts:20]: are these eslint-disabled lines still required?

**Review by MarcMcIntosh — COMMENTED** [review]
Is there a difference between `eg`, `.e.g` and `i.e` ?

**Comment by chengoconnell** [comment]
> Is there a difference between `eg`, `.e.g` and `i.e` ?

ah eg = typo
e.g. = for example
i.e. = explanation 

i'll fix the typo good spot!

File: packages/slice-machine/lib/forms/defaults.ts (hunk 1 above)
Comment (chengoconnell) [reply packages/slice-machine/lib/forms/defaults.ts:20]: yup they error if i delete them

Comment (MarcMcIntosh) [reply packages/slice-machine/lib/forms/defaults.ts:20]: balls :/

**Review by MarcMcIntosh — APPROVED** [review]

**Pull request merged**